File: NetBeansProjects/WebApplication1/src/java/com/epic/canvascontrollibrary/CCLImageFaderProps.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.epic.canvascontrollibrary;

import java.util.ArrayList;
import java.util.List;

/**
 *
 * @author Gilgamesh
 */
    public class CCLImageFaderProps
    {
        public String CanvasID;
        public String WindowID;
        public String X;
        public String Y;
        public String Width;
        public String Height;
        public List<Object> ImageURLs;
        public String FadeStartValue;
        public String FadeEndValue;
        public String FadeStepValue;
        public String HoldForTicks;
        public String ClickFunction;
        public String HoldCountDown;
        public String CurrentImageIndex;
        public String CurrentGlobalAlphaValue;
        public String OverlayImages;
        public String TabStopIndex;

        CCLImageFaderProps()
        {
            ImageURLs = new ArrayList<Object>();
        }
    }

File: ASP.NET MVC Web Api New Solution/CanvasControlLibrary/CanvasControlLibrary/Models/CCLImageFaderProps.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations.Schema;

namespace CanvasControlLibrary.Models
{
    public class CCLImageFaderProps
    {
        public string CanvasID { get; set; }
        public string WindowID { get; set; }
        public int X { get; set; }
        public int Y { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
        public List<string> ImageURLs { get; set; }
        public int FadeStartValue { get; set; }
        public int FadeEndValue { get; set; }
        public int FadeStepValue { get; set; }
        public int HoldForTicks { get; set; }
        public int HoldCountDown { get; set; }
        public int CurrentImageIndex { get; set; }
        public float CurrentGlobalAlphaValue { get; set; }
        public int OverlayImages { get; set; }
        public object Tag { get; set; }
        public int TabStopIndex { get; set; }

        public CCLImageFaderProps()
        {
            ImageURLs = new List<string>();
        }
    }
}